docs/overview/office-add-in-code-samples.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Office Add-in code samples
+description: 'Office Add-in code samples'
+ms.date: 09/09/2021
+localization_priority: Normal
+---
+
+# Office Add-in code samples
+
+The code samples listed in this article are written to help you learn how to use various features when developing Office Add-ins.
+
+## Outlook
+
+| Name                | Description         |
+|:--------------------|:--------------------|
```

Comment:
David, when I view the readme and click the all the links, I get page not found.

Reply:
Yeah. the air-gap requirements (shortened docs.microsoft.com links) don't work well with the [staging site](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-in-code-samples?branch=pr-en-us-2931) since the linked repo doesn't have the same branch name. My trick for testing links like this is to open the link in a new tab, then remove the `review` part of the URL. So, in this case
"https://review.docs.microsoft.com/en-us/samples/officedev/pnp-officeaddins/outlook-add-in-tag-external-recipients?branch=pr-en-us-2931"
becomes
"https://docs.microsoft.com/en-us/samples/officedev/pnp-officeaddins/outlook-add-in-tag-external-recipients?branch=pr-en-us-2931"
(That ignores the branch attribute of the URL query because it takes you to the published site.)

Reply:
Yup I verified the links work as expected when constructed that way. Thanks for verifying.